pyanaconda/core/constants.py

```diff
@@ -467,3 +467,11 @@ class PayloadRequirementType(Enum):
 
 # Path to the System Purpose configuration file on a system.
 RHSM_SYSPURPOSE_FILE_PATH = "/etc/rhsm/syspurpose/syspurpose.json"
+
+
+# Asynchronous subscription state tracking
+class SubscriptionPhase(Enum):
```

Comment:
I would put this enum to the same file as `register_and_subscribe` and `unregister`.

Reply:
Yeah, we can move it.